ra_aid/tools/expert.py

```
@@ -13,6 +15,46 @@
+    Returns:
+        str: Name of the first model with specified tier, or empty string if none found
+    """
+    for model_name, config in reasoning_tiers.items():
+        if config["tier"] == tier:
+            return model_name
+    return ""
+
+
+def get_best_expert_model_by_reasoning_tier_capabilities_and_specialties(
+    tier: int = ReasoningTier.EXPERT,
+    capabilities: List[str] = [],
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)